socorro/siglists/prefix_signature_re.txt

```diff
@@ -19,8 +19,10 @@ CERT_.*
 CFRelease
 _chkstk
 CleanupPerAppKey
+ConditionVariableFallback::wait.*
```

Comment:
I would do ``ConditionVariableFallback::wait(.*)`` in order to not catch unexpected things, like ``waitForSmth`` (if that ever exists).